programs/candy-machine-core/program/Cargo.toml

```diff
@@ -21,7 +21,7 @@ anchor-lang = "0.28.0"
 arrayref = "0.3.6"
 mpl-token-metadata = "3.2.1"
 mpl-utils = { version = "0.3", default-features = false }
-mpl-core = { version = "0.4.3" }
+mpl-core = { version = "0.6.1" }
```

Comment:
FYI I release 0.7.0 yesterday.  I thought you were having a CPI issue with assetBurn guard and 0.6.1 is from 16 days ago.